packages/dd-trace/src/llmobs/writers/spans/base.js

```diff
@@ -18,11 +19,18 @@ class LLMObsSpanWriter extends BaseWriter {
 
   append (event) {
     const eventSizeBytes = Buffer.from(JSON.stringify(event)).byteLength
+    telemetry.submitLLMObsRawSpanSize(event, eventSizeBytes)
+
+    let processedEventSizeBytes = eventSizeBytes
```

Comment:
this made me realize we should actually be passing this `processedEventSizeBytes` event size to the `super.append` call below instead of `eventSizeBytes`, and we probs should've had this re-computation already 😱 idt it's been an issue at all but is something we should fix

 i can make this change in a PR i have for other writers-based changes!